exercises/grade-school2/canonical-data.json

```diff
@@ -0,0 +1,155 @@
+{
+    "exercise": "grade-school2",
+    "comments": [
+        "Given students' names along with the grade that they are in, ",
+        "create a roster for the school."
+    ],
+    "cases": [
+        {
+          "uuid": "806b873f-13ca-4b19-9161-e3e11bdd15ef",
+          "description": "Roster is empty when no student is added",
+          "property": "roster",
+          "input": {
+            "students": []
+          },
+          "expected": []
+        },
+        {
+          "uuid": "9337267f-7793-4b90-9b4a-8e3978408824",
+          "description": "Add a student",
+          "property": "add",
+          "input": {
+            "students": [["Aimee", 2]]
+          },
+          "expected": true
+        },
+        {
+          "uuid": "aff83e6e-6bdc-4753-9f98-142d19e3f5c4",
+          "description": "Student is added to the roster",
+          "property": "roster",
+          "input": {
+            "students": [["Aimee", 2]]
+          },
+          "expected": ["Aimee"]
+        },
+        {
+          "uuid": "73c3ca75-0c16-40d7-82f5-ed8fe17a8e4a",
+          "description": "Adding multiple students in same the grade",
+          "property": "add",
+          "input": {
+            "students": [["Blair", 2], ["James", 2], ["Paul", 2]]
+          },
+          "expected": [true, true, true]
+        },
+        {
+          "uuid": "c398a433-7a4e-4d80-b379-938944c68a64",
+          "description": "Multiple students in the same grade are added to the roster",
+          "property": "roster",
+          "input": {
+            "students": [["Blair", 2], ["James", 2], ["Paul", 2]]
+          },
+          "expected": ["Blair", "James", "Paul"]
+        },
+        {
+          "uuid": "87c871c1-6bde-4413-9c44-73d59a259d83",
+          "description": "Cannot add student to same grade more than once",
+          "property": "add",
+          "input": {
+            "students": [["Aimee", 2], ["Aimee", 2]]
+          },
+          "expected": [true, false]
+        },
+        {
+          "uuid": "d7982c4f-1602-49f6-a651-620f2614243a",
+          "description": "Student not added to same grade more than once",
+          "property": "roster",
+          "input": {
+            "students": [["Aimee", 2], ["Aimee", 2]]
+          },
+          "expected": ["Aimee"]
+        },
+        {
+          "uuid": "e70d5d8f-43a9-41fd-94a4-1ea0fa338056",
+          "description": "Adding students in multiple grades",
+          "property": "add",
+          "input": {
+            "students": [["Chelse", 3], ["Logan", 7]]
```

Comment:
Although it does not affect correctness since the name does not appear in the output, if the other instance of Chelse is changed, it may be prudent to change this one as well for consistency.